README.md

```diff
@@ -29,9 +29,20 @@ yarn add flowchart-vue
         <button type="button" @click="$refs.chart.save()">
             Save
         </button>
-        <flowchart :nodes="nodes" :connections="connections" @editnode="handleEditNode"
-                    @dblclick="handleDblClick" @editconnection="handleEditConnection" 
-                    @save="handleChartSave" ref="chart">
+        <button type="button" v-if="showRemovingConfirmation" @click="confirmRemoving">
```

Comment:
Can't understand the meaning of `@click="confirmRemoving"`, maybe it should be `initRemovingConfirmation` or `showRemovingConfirmation = true`?

Reply:
Actually it is confirmation - you decides to agree that you are sure you want to delete elements.
Wrong value was in `showRemovingConfirmation`.

Reply:
Solved.